src/store/index.js

```diff
@@ -116,10 +118,33 @@ const store = new Vuex.Store({
       }
     },
 
+    async googleSigninoAuth() {
```

Comment:
I don't see anything that modifies the state here. This should be moved to firebase.js